### ISSUE TITLE
Lighthouse bug in a11y test

### ISSUE THREAD

**Issue by travisdowns** [travisdowns/travisdowns.github.io#449]
For sorting.html, the lighthouse a11y test fails and returns null.

**Comment by travisdowns** [comment]
Filed as https://github.com/GoogleChrome/lighthouse/issues/13118.
